flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_LocalVarUpdater.js

```diff
@@ -272,5 +278,14 @@ class A32NX_LocalVarUpdater {
         return this.opacityMcdu;
     }
 
+    cvrGroundControl(deltaTime, identifier, currentValue) {
+        if (currentValue) {
+            const isEngineOneRunning = SimVar.GetSimVarValue("ENG COMBUSTION:1", "bool");
+            const isEngineTwoRunning = SimVar.GetSimVarValue("ENG COMBUSTION:2", "bool");
+            return !isEngineOneRunning && !isEngineTwoRunning;
+        }
+        return currentValue;
```

Comment:
When will `currentValue` be returned?

Reply:
Standby, I am re-implementing this with a better reference. ;)